### PR TITLE
Example and Binding updates for new image handling

### PR DESCRIPTION
**Pull request by pgrawehr**
Update examples related to image handling. This PR depends on #2069.

Includes new and updated bindings:
- Ili9342 QVGA display (moved namespace to Ili934x, because very similar to Ili9341)
- Ili9341 QVGA display driver updated
- Axp192 Power controller, including device-dependent abstraction for M5Tough
- Fixed analog reporting in Arduino binding for never boards

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/iot/pull/2084)

### CODE REVIEW

File: src/devices/Ili934x/samples/images/OpenMenu.png
Comment (krwq) [inline src/devices/Ili934x/samples/images/OpenMenu.png:1]: What's the source of these pictures?

File: src/devices/Common/Iot/Device/Graphics/MouseClickSimulatorUInput.uinput.cs [hunk 1]
```diff
@@ -0,0 +1,210 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```
Comment (krwq) [inline src/devices/Common/Iot/Device/Graphics/MouseClickSimulatorUInput.uinput.cs:1]: this feels like it should be in separate PR and is a bit out of scope for this repo

File: src/devices/Common/Interop/Unix/Interop.uinput.cs [hunk 2]
```diff
@@ -0,0 +1,201 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```
Comment (krwq) [inline src/devices/Common/Interop/Unix/Interop.uinput.cs:1]: is this generated? why do we need all these?

File: src/devices/Common/Interop/Unix/Interop.X11.cs [hunk 3]
```diff
@@ -0,0 +1,429 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
```
Comment (krwq) [inline src/devices/Common/Interop/Unix/Interop.X11.cs:4]: same question, why do we need X11 interop added as part of image library refactoring?

File: src/devices/Axp192/M5ToughPowerControl.cs [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+            }
+        }
+
+        private void SendCpuToSleep()
+        {
+            // Sends the board to sleep mode
+            TimeSpan sleepDelay = TimeSpan.FromSeconds(10);
+            const int wakeupPin = 39; // The screen interrupt is connected to this pin
+
+            if (!(_board is ArduinoBoard ab))
```
Comment (krwq) [inline src/devices/Axp192/M5ToughPowerControl.cs:183]: why does Axp192 need to depend on arduino board? this complicates porting to things like nano or using this library standalone

Comment (pgrawehr) [reply src/devices/Axp192/M5ToughPowerControl.cs:183]: It's not the AXP that depends on the board, but the M5Though device. This is a device containing an AXP, an ESP32, a display and several other components, and for this library, Arduino is currently the only way to talk to it. In here, we need to send some special commands to the CPU.

**Comment by pgrawehr** [comment]
Closing, as now integrated in various other PRs.
